pkg/util/timer/after_test.go

```diff
@@ -23,8 +23,31 @@ package timer
 import (
 	"testing"
 	"time"
+
+	"github.com/arangodb/kube-arangodb/pkg/util/tests/memory"
 )
 
 func Test_After(t *testing.T) {
 	<-After(250 * time.Millisecond)
 }
+
+func afterManyTimes(duration time.Duration, times int) {
+	for i := 0; i < times; i++ {
+		<-After(duration)
+	}
+}
+
+func Test_MemoryLeak_After(t *testing.T) {
+	t.Run("Initial wrap", func(t *testing.T) {
+		afterManyTimes(time.Nanosecond, 1024^2)
+	})
+
+	t.Run("Memory should me stable", func(t *testing.T) {
+		start := memory.Sys()
+		for i := 0; i < 256; i++ {
+			afterManyTimes(time.Nanosecond, 1024^2)
+			now := memory.Sys()
+			t.Logf("Memory grow by %d", now-start)
```

Comment:
Maybe test should fail if memory has grown more than expected?

Reply:
We cant predict it - it is used to check via logs if there is grow. I can remove this test due to fact it is fixed